code/__DEFINES/maths.dm

```diff
@@ -81,6 +81,10 @@
 // amount=0.5 returns the mean of a and b.
 #define LERP(a, b, amount) ( amount ? ((a) + ((b) - (a)) * (amount)) : a )
 
+// Moves the current number towards the target number by amount.
+// It won't overshoot
```

Comment:
`/**` comment pattern?

Reply:
Nothing seems to use this anyway, i'll just get rid of it